services/core-api/app/api/mines/permits/permit_conditions/resources/permit_conditions_resource.py

```diff
@@ -96,4 +96,22 @@ def delete(self, mine_guid, permit_guid, permit_amendment_guid, permit_condition
 
         permit_condition.save()
 
+        if permit_condition.parent_permit_condition_id is not None:
+            parent = PermitConditions.find_by_permit_condition_id(
+                permit_condition.parent_permit_condition_id)
+            sorted_conditions = sorted(parent.sub_conditions, key=lambda x: x.display_order)
+            for i, condition in enumerate(sorted_conditions):
+                condition.display_order = i + 1
+            parent.save()
+        else:
+            conditions = PermitConditions.find_all_by_permit_amendment_id(
+                permit_condition.permit_amendment_id)
```

Comment:
Might want to load permit_amendment then get conditions member. but i suppose they are equivalent.